scripts/ganache.sh

```diff
@@ -9,23 +9,28 @@ helpFunction() {
 
 start() {
 	. scripts/env.sh
-	
+
 	if [ -z "${ORIGARGS[1]}" ]
 	then
 		npx ganache ethereum -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123"
 	else
 		echo "Starting ganache ..."
-		docker run --detach --publish 8545:8545 trufflesuite/ganache:latest -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123"
+		echo "npx ganache ethereum -m \"$WEB3_SYSTEM_TEST_MNEMONIC\" -a 5 -p $WEB3_SYSTEM_TEST_PORT &"
+        npx ganache ethereum -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT >/dev/null --wallet.passphrase "123" &
+
+#		docker run --detach --publish 8545:8545 trufflesuite/ganache:latest -m "$WEB3_SYSTEM_TEST_MNEMONIC" -a 5 -p $WEB3_SYSTEM_TEST_PORT --wallet.passphrase "123"
```

Comment:
Once you have the tests passing, we should revert this change and see if it works with docker. I believe making this change back to `npx` will cause the black box tests to fail

Reply:
reverted

Reply:
We should probably remove the commented code if we aren't going to use it. Additionally, we should consider replacing the use of `npx` when not started in `background` mode. Lastly, the `stop` command will only work for Docker, so we need to either remove the use of `npx` or update `stop` to know whether to kill the ganache process or stop the Docker container

Reply:
fixed. thanks!